README.md

```diff
@@ -1,240 +1,60 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
```

Comment:
- [ ] You have submitted a project, that uses GitHub Flow but you are supposed to use Gitflow. To use Gitflow you follow these rules :arrow_down:
- You make a development branch from the main/master branch :+1:
- Then you make your development branch default :blush:
- At last, whenever you wanna make a change or project, or milestone then you make your feature branch from the development branch and after making that change you would merge that feature branch to development branch :+1: